docs/src/main/doc/IntegratingDdfSpatial.adoc

```diff
@@ -0,0 +1,2089 @@
+= Integrating {branding} Spatial App
+Copyright (c) Codice Foundation
+2.7-SNAPSHOT
```

Comment:
We could look into maybe using maven properties for version.  That would be pretty awesome.